bootstrap.sh

```diff
@@ -154,14 +154,14 @@ if [ -z "$MYSQL_FLAVOR" ]; then
 fi
 case "$MYSQL_FLAVOR" in
   "MySQL56")
```

Comment:
In general I didn't want the "flavor" concept to be related to versions. The main point of flavors is to deal with forks of MySQL that diverge in significant ways, such as in GTID implementations. For example, there should be one flavor for Oracle's lineage and one for MariaDB since they've diverged to this extent.

The fact that the "MySQL56" flavor has a number in it is a mistake. I named it that because I thought "MySQL" would be too generic and I couldn't think of a better name - "OracleMySQL"?

The reason test.go distinguishes between 5.6 and 5.7 is because it needs to pick a specific Docker image, and we now maintain Docker images for multiple versions within a flavor. This is another mistake in naming; I shouldn't have called the test.go flag "flavor" because it's a different dimension than MYSQL_FLAVOR.